backend/src/controllers/cars.controller.ts

```diff
@@ -1,8 +1,17 @@
-import { TCarUpdate } from "../interfaces/car.interfaces"
+import { TCarResponse, TCarUpdate } from "../interfaces/car.interfaces"
 import { deleteContactsService } from "../services/cars/deleteCars.service";
+import { getByIdCarsService } from "../services/cars/getByIdCars.service";
 import { updateCarsService } from "../services/cars/updateCars.service"
 import { Response, Request } from "express";
 
+const getByIdCarsController = async (req: Request, res: Response): Promise<Response> => {
+
+    const carId: number = Number(req.params.id)
+
+    const car:TCarResponse= await getByIdCarsService(carId)
+
+    return res.json(car)
```

Comment:
missing .status(200)

Reply:
Added the missing .status(200) as suggested.